config/samples/kamaji_v1alpha1_datastore_postgresql.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   driver: PostgreSQL
   endpoints:
-    - postgresql-rw.kamaji-system.svc:5432
+    - postgresql-rw.kamaji-system.svc.cluster.local:5432
```

Comment:
@prometherion `postgresql-rw.kamaji-system.svc.cluster.local` is not recognised as a valid CN in certificates. Should revert back?